pkg/cache/cache.go

```diff
@@ -0,0 +1,90 @@
+package cache
+
+import (
+	"path"
+	"strings"
+
+	zlog "github.com/anuvu/zot/pkg/log"
+	"github.com/boltdb/bolt"
+)
+
+const (
+	BlobsCache = "blobs"
+)
+
+type Cache struct {
+	db  *bolt.DB
+	log zlog.Logger
+}
+
+// Blob is a blob record
+type Blob struct {
+	Path string
+}
+
+func NewCache(rootDir string, name string, log zlog.Logger) *Cache {
+	db, err := bolt.Open(path.Join(rootDir, name+".db"), 0600, nil)
+	if err != nil {
+		panic(err)
```

Comment:
Let's return an error here (and elsewhere) instead; that way calling code can decide what to do instead of just panic.